python/py-fqdn/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-fqdn
+version             1.5.1
+revision            0
+categories-append   devel
+license             MPL-2
+
+supported_archs     noarch
+platforms           {darwin any}
+
+python.versions     38 39 310 311 312
+
+maintainers         nomaintainer
+
+description         Validates fully-qualified domain names against RFC 1123
+long_description    {*}${description}
+
+homepage            https://github.com/ypcrts/fqdn
+
+checksums           rmd160  b81eaad9b61dac0838c81cf652a30a76c260a2ba \
+                    sha256  105ed3677e767fb5ca086a0c1f4bb66ebc3c100be518f0e0d755d9eae164d89f \
+                    size    6015
+
+if {${name} ne ${subport}} {
+    depends_lib-append  port:py${python.version}-cached-property
```

Comment:
Probably not needed (only for python<3.8).
https://github.com/ypcrts/fqdn/blob/3a4feda3414bde7b3386e995ed1d2ed36607d4cf/setup.py#L17